__tests__/app.test.js

```diff
@@ -187,4 +187,68 @@ describe("app", () => {
         });
     });
   });
+
+  describe("POST /api/reviews/:review_id/comments", () => {
+    // request body accepts  an object with username and body
+    // responds with POSTed comment
+    test("201: responds with newly created comment", () => {
+      // const requestBody = {
+      //   username: "TheCakeIsALie",
+      //   body: "A fun afternoon! Definitely recommend!!!",
+      // };
+      return request(app)
+        .post("/api/reviews/2/comments")
+        .expect(201)
+        .send({
+          username: "philippaclaire9",
+          body: "A fun afternoon! Definitely recommend!!!"
+        })
+        .then(({ body }) => {
+          expect(body.comment).toMatchObject({
+            comment_id: 7,
+            body: "A fun afternoon! Definitely recommend!!!",
+            votes: 0,
+            author: "philippaclaire9",
+            review_id: 2,
+            created_at: expect.any(String),
+          });
+        });
+    });
+    test("400: GET invalid review_id endpoint when trying to POST comments", () => {
+      return request(app)
+        .post("/api/reviews/cake/comments")
+        .send({
+          username: "philippaclaire9",
+          body: "A fun afternoon! Definitely recommend!!!",
+        })
+        .expect(400)
```

Comment:
lets add another 400 test for missing required field(s), e.g. no username or body properties in the post